inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -493,12 +488,16 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     // Try to create operation from loaded opsets
     if (!ngraphNode && opsets.count(params.version)) {
         auto opset = opsets.at(params.version);
+        std::string type = params.type;
 
         if (!opset.contains_type(params.type)) {
-            THROW_IE_EXCEPTION << "Opset " << params.version << " doesn't contain the operation with type: " << params.type;
+            std::transform(type.begin() + 1, type.end(), type.begin() + 1, ::tolower);
```

Comment:
Why do we need to transform type to lower case?
Do you have examples?

Reply:
This transformation is for Elu op. In current functional test (ReadELUNetwork) type of the op for v10 is "ELU". That's why I added to lower case transformation. Looks like this notation is wrong, because from model optimizer I got "Elu" for v10. I will rename the op in test  and remove transformation.

Reply:
Corrected.